socket-server/cookie-authorize.js

```diff
@@ -0,0 +1,26 @@
+var cookie = require('cookie');
+var jwt = require('jsonwebtoken');
+
+var config = require('config.js');
+
+module.exports = function authorize(socket) {
```

Comment:
As above, if you use socketio-jwt or some other alternative you can put the auth stuff in `socker-server/index.js` without making it messy. It will also give you easier management of failures.